sqrl-calcite/src/main/java/com/datasqrl/plan/validate/ScriptPlanner.java

```diff
@@ -155,35 +155,29 @@ public Void visit(SqrlCreateDefinition statement, Void context) {
     NamePath namePath = NamePath.of(statement.getName().names.toArray(String[]::new));
     String logId = statement.getName().names.get(0);
 
-    Map map = new HashMap();
-    map.put("primary-key", List.of("_uuid"));
-    map.put("timestamp-type", "LOG_TIME");
```

Comment:
Is kafka as a log source broken now? I see the timestamp-type is no longer being specified

Reply:
No, it is not broken, usecasetests are passing. I pass these values to the logmanager.